README.md

```diff
@@ -17,11 +32,438 @@ Xcode 16 or later.
 
 ## Installation
 
-The SDK is distributed as a Swift package and can hence be installed using Xcode or by adding it as a dependency in your package’s `Package.swift`. We’ll add detailed instructions when we release the first version of the SDK.
+The SDK is distributed as a Swift package and can hence be installed using Xcode or by adding it as a dependency in your package’s `Package.swift`:
+
+```swift
+.package(url: "https://github.com/ably/ably-chat-swift", from: "0.1.0")
+```
+
+## Supported chat features
+
+This project is under development so we will be incrementally adding new features. At this stage, you'll find APIs for the following chat
+features:
+
+- Chat rooms for 1:1, 1:many, many:1 and many:many participation.
+- Sending and receiving chat messages.
+- Online status aka presence of chat participants.
+- Chat room occupancy, i.e total number of connections and presence members.
+- Typing indicators
+- Room-level reactions (ephemeral at this stage)
+
+If there are other features you'd like us to prioritize, please [let us know](https://forms.gle/mBw9M53NYuCBLFpMA).
+
+## Usage
+
+You will need the following prerequisites:
+
+- An Ably account
+    - You can [sign up](https://ably.com/signup) to the generous free tier.
+
+- An Ably API key
+    - Use the default or create a new API key in an app within
+      your [Ably account dashboard](https://ably.com/dashboard).
+    - Make sure your API key has the
+      following [capabilities](https://ably.com/docs/auth/capabilities): `publish`, `subscribe`, `presence`, `history` and
+      `channel-metadata`.
+
+To instantiate the Chat SDK, create an [Ably client](https://ably.com/docs/getting-started/setup) and pass it into the Chat constructor:
+
+```swift
```

Comment:
How about showing that the user needs to import `Ably` and `AblyChat`?

Also, how do transitive dependencies work in Swift — will the user be able to import `Ably` simply because it's a dependency of `AblyChat`, or do they need to explicitly specify ably-cocoa in their `Package.swift`?

Reply:
Example app imports ably-cocoa as well as other `AblyChat` dependencies without need of explicit adding it. Re `import` it's a good point. But can we add a constructor that will allow user only import `AblyChat`? Such as
`init(key: String, clientId: String, clientOptions: ClientOptions?)`

Reply:
Updated

Reply:
> But can we add a DefaultChatClient constructor that will allow user only import AblyChat?

I don't know why we chose to require the user to always explicitly create their own core SDK client; my guess would be that there are use cases for doing so and we decided we didn’t want to have multiple ways of instantiating a wrapper SDK 🤷. Anyway I think doing so is outside the scope of this PR.

Reply:
Do you know the answer to this question?

> Also, how do transitive dependencies work in Swift — will the user be able to import Ably simply because it's a dependency of AblyChat, or do they need to explicitly specify ably-cocoa in their Package.swift?